src/Functions/Public/relocation/New-vRAOnboardingResource.ps1

```diff
@@ -141,7 +141,7 @@
                 $URI = "/relocation/onboarding/resource"
                 $OnboardingResource = Invoke-vRARestMethod -Method POST -URI $URI -Body $Body -Verbose:$VerbosePreference
 
-                CalculateOutput
+                CalculateOutput($OnboardingResource)
```

Comment:
Should the parameter be positional here and not in ()?

Reply:
Should the parameter be positional here and not in ()?